web/src/components/tasks/CheckElevatorRegistrationStatus.tsx

```diff
@@ -71,7 +71,7 @@ export const CheckElevatorRegistrationStatus = (props: Props): ReactElement => {
       {Config.elevatorRegistrationSearchTask.registrationSearchPrompt}
       <form onSubmit={onSubmit} className={"padding-top-1"}>
         <div className="margin-bottom-2">
-          <label htmlFor="address-1">{Config.licenseSearchTask.address1Label}</label>
+          <label htmlFor="address-1">{Config.elevatorRegistrationSearchTask.address1Label}</label>
```

Comment:
I missed these in my previous PR - should be drawing from this task instead of licenseSearch